content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -17,84 +20,154 @@ locations.
-total number of sent emails.
+.. example::
+   A monthly newsletter has been sent out, and 6.9% of the sent emails were bounced.
 
-A high received rate can indicate that the mailing list used is up to date and the sender
-authentication is trusted by email providers.
+   .. image:: analyze_metrics/newsletter-metrics.png
+      :align: center
+      :alt: The metrics smart buttons of the newsletter.
 
-A low received rate may indicate issues either with the mailing list used for the mailing or with
-the sender authentication. View the :ref:`email-marketing/deliverability-issues` section for more
-information.
+   To see what these bounced recipients have in common, the records are grouped using a custom group
+   targeting :guilabel:`Mailing Lists`, which groups all records by the mailing lists they are on.
+   The records are then filtered using a custom filter with the rule `Created on >= 07/01/2024
+   00:00:00`, when the mailing list was last checked. This filter only includes recipients
+   that have been created on, or after, July 1st, 2024, in the report.
 
-Bounced rate
-------------
+      .. image:: analyze_metrics/metrics-filter.png
+         :align: center
+         :alt: The custom filter creation form.
 
-The *Bounced* rate measures the percentage of emails that were unsuccessfully delivered, against the
-total number of sent emails.
+   Using these configurations, it is evident that all the recipients with bounced emails were added
+   after the list was last checked. Looking closer at the domains, it can be seen that each
+   recipient has a malformed email domain (i.e: @yaoo.com instead of @yahoo.com), likely due to a
+   manual entry error while updating the database.
 
-A high bounce rate could indicate issues either with the mailing list used for the mailing or with
-the sender authentication.
+   .. image:: analyze_metrics/malformed-addresses.png
+      :align: center
+      :alt: A list of bounced email addresses with malformed email domains.
 
-A low bounce rate may indicate that the mailing list used is up to date and the sender
-authentication is trusted by email providers. View the :ref:`email-marketing/deliverability-issues`
-section for more information.
+.. seealso::
+  View :doc:`../../essentials/search` for more information about making custom groups and
+  filters.
 
 .. _email-marketing/deliverability-issues:
 
+Mass mailing analysis
+=====================
+
+It is also possible to analyze the success between mailing campaigns by creating a *Mass Mailing
+Analysis* report. To begin, navigate to :menuselection:`Email Marketing app --> Reporting --> Mass
+Mailing Analysis`.
+
+A dashboard appears displaying a bar chart containing each mailing campaign. By default, *Sent* is
+selected, displaying the number of sent records on the y-axis. To change the measure, click the
+:guilabel:`Measures` button, and select the desired measure from the drop-down menu.
+
+.. example::
```

Comment:
GREAT example! 
Consider writing a tutorial just on this topic around open rate optimization 🤙